packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -277,6 +281,22 @@ class WebView extends StatefulWidget {
   /// [WebViewController.evaluateJavascript] can assume this.
   final PageFinishedCallback onPageFinished;
 
+  /// Invoked when a web resource has failed to load.
+  ///
+  /// This can be called for any resource (iframe, image, etc.), not just for
+  /// the main page.
+  ///
+  /// On Android, the error code will be a constant from
+  /// [WebViewClient](https://developer.android.com/reference/android/webkit/WebViewClient#summary).
+  /// (e.g. The error code of `WebViewClient.ERROR_AUTHENTICATION` on Android
+  /// translates to `AUTHENTICATION` in Dart.
+  ///
+  /// On iOS, the error code will represent the "domain" and the "code" for an
```

Comment:
My preference would be to avoid having different semantic meaning to the same field on different platforms.
Instead I'd have different fields with a single semantic meaning which may not be set on all platforms.

Reply:
done. I added a `domain` only for iOS and a `errorType` only for Android.